cyclops-ui/src/components/pages/ModuleDetails.tsx

```diff
@@ -378,6 +378,11 @@ const ModuleDetails = () => {
           <Deployment name={resource.name} namespace={resource.namespace} />
         );
         break;
+      case "CronJob":
+        resourceDetails = (
+          <CronJob name={resource.name} namespace={resource.namespace} />
```

Comment:
You will have to import the CronJob component you created. Check lines 31 to 37 in this file